src/Lib/Core/Email.hs

```diff
@@ -17,21 +16,3 @@ newtype Email = Email { unEmail :: Text }
     deriving stock (Show, Generic)
     deriving newtype (Eq, Ord, Hashable, FromField, ToField)
     deriving anyclass (FromJSON, ToJSON, ElmType, ToSchema, ToParamSchema)
```

Comment:
This code doesn't compile. Since you removed `swagger` from dependencies you can no longer have `ToSchema` and `ToParamSchema` type classes.